jenkins/Jenkinsfile.hourly

```diff
@@ -0,0 +1,39 @@
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds()
+    buildDiscarder(
+      logRotator(
+        artifactDaysToKeepStr: '',
+        artifactNumToKeepStr: '',
+        daysToKeepStr: '',
+        numToKeepStr: '100'))
+    rateLimitBuilds(throttle: [count: 1, durationName: 'hour', userBoost: true])
+  }
+  
+  parameters {
+    booleanParam name: 'PUBLISH', defaultValue: true, description: 'Publish the build to S3 and sentry.'
+    string name: 'OS_FILTER', defaultValue: 'all', description: 'Pattern to limit builds by matching OS'
+    string name: 'ARCH_FILTER', defaultValue: 'all', description: 'Pattern to limit builds by matching ARCH'
+    string name: 'SLACK_CHANNEL', defaultValue: '#ide-builds', description: 'Slack channel to publish build message.', trim: true
+  }
+
+  stages {
+    stage('Trigger Hourly Build') {
```

Comment:
Is `SCMTrigger` still needed for the hourly build? Or is it already being triggered on commit?

Reply:
It's already been triggered on commit. The other Jenkinsfiles use `SCMTrigger` to decide if a stage should run or not. Since the hourly triggers on every commit we don't need it here